Unit tests broken at root level

I did a fresh clone of the repo and tried to `npm install` and `npm test` and got the following error:

According to our README, `npm test` should work at the root level with no changes.  We need to either fix this or update the readme.


```
[jsprague@localhost spaship]$ npm test

> @spaship/spaship@0.3.3 test /home/jsprague/projects/spaship
> CI=true npm run lerna run test


> @spaship/spaship@0.3.3 lerna /home/jsprague/projects/spaship
> lerna "run" "test"

lerna notice cli v3.22.0
lerna info ci enabled
lerna info Executing command in 6 packages: "npm run test"
lerna ERR! npm run test exited 1 in '@spaship/manager'
lerna ERR! npm run test stdout:

> @spaship/manager@0.10.0 test /home/jsprague/projects/spaship/packages/manager
> react-scripts test


lerna ERR! npm run test stderr:

There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "eslint": "^6.6.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of eslint was detected higher up in the tree:

  /home/jsprague/projects/spaship/node_modules/eslint (version: 7.1.0) 

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "eslint" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /home/jsprague/projects/spaship/node_modules/eslint is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls eslint in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed eslint.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @spaship/manager@0.10.0 test: `react-scripts test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @spaship/manager@0.10.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jsprague/.npm/_logs/2020-05-27T20_26_13_165Z-debug.log

lerna ERR! npm run test exited 1 in '@spaship/manager'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @spaship/spaship@0.3.3 lerna: `lerna "run" "test"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @spaship/spaship@0.3.3 lerna script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jsprague/.npm/_logs/2020-05-27T20_26_13_185Z-debug.log
npm ERR! Test failed.  See above for more details.
[jsprague@localhost spaship]$ 

```

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "private": true,
   "devDependencies": {
     "codecov": "3.7.0",
-    "eslint": "7.1.0",
+    "eslint": "^6.6.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.3",
     "husky": "4.2.5",

